packages/graphback/src/layers/service/GraphbackCRUDService.ts

```diff
@@ -0,0 +1,23 @@
+import { AdvancedFilter } from '../data/GraphbackDataProvider';
+
+
+/**
+ * TODO
+ */
+// tslint:disable-next-line: no-any
+export interface GraphbackCRUDService<Type = any, GraphbackContext = any> {
+    // FIXME Pagination support
+    // CRUD
+    createObject(name: string, data: Type, context?: GraphbackContext): Promise<Type>;
+    updateObject(name: string, id: string, data: Type, context?: GraphbackContext): Promise<Type>;
+    deleteObject(name: string, id: string, context?: GraphbackContext): Promise<string>;
```

Comment:
If I would like to extend the GraphbackCRUDService to an Optimistic Locking Service that checks the model version before performing any mutation it would be impossible because with the current interface I couldn't read also the version args. I know it's probably a corner case but maybe should take it into consideration.

Reply:
I've created a dummy example: https://github.com/aerogear/graphback/blob/alternative-runtime-resolver/packages/graphback/src/generators/resolvers/layered/LayeredRuntimeResolverGen.ts

Reply:
Target is to pass more to delete method than just id in case we need to make sure that delete is possible